cli/src/commands/init/index.ts

```diff
@@ -158,6 +159,8 @@ export default class Init extends BaseCommand<typeof Init> {
       await this.deploySchema(workspace, region, database, branch, schema);
     }
 
+    // Run pull to retrieve remote migrations, remove any local migrations, and generate code
```

Comment:
This comment made me think pull generates code? 

If it does, should we remove the codegen line?

Reply:
It does but the config is not saved yet, so we need to call them both.